Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -760,6 +770,7 @@ class MutableComputedValues final : public ComputedValues {
     void set_justify_items(CSS::JustifyItems value) { m_noninherited.justify_items = value; }
     void set_justify_self(CSS::JustifySelf value) { m_noninherited.justify_self = value; }
     void set_box_shadow(Vector<ShadowData>&& value) { m_noninherited.box_shadow = move(value); }
+    void set_rotate(CSS::Transformation value) { m_noninherited.rotate = value; }
```

Comment:
We also want a getter for this in `ComputedValues`, so that later code can read it.